docs/content/preview/drivers-orms/php/laravel.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Laravel ORM
+headerTitle: Use an ORM
+linkTitle: Use an ORM
+description: Laravel ORM support for YugabyteDB
+headcontent: Laravel ORM support for YugabyteDB
+image: /images/section_icons/sample-data/s_s1-sampledata-3x.png
+menu:
+  preview:
+    identifier: laravel-orm
+    parent: php-drivers
+    weight: 500
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li >
+    <a href="../laravel/" class="nav-link active">
+      Laravel ORM
+    </a>
+  </li>
+</ul>
+
+[Laravel](https://laravel.com/docs/10.x/readme) is a php web application framework with expressive, elegant syntax. Along with many features required for web development, Laravel includes [Eloquent](https://laravel.com/docs/10.x/eloquent#introduction) ORM which provides a robust and intuitive database ORM.
+
+YugabyteDB YSQL API has full compatibility with Laravel's Eloquent ORM for data persistence in php applications.
+
+Learn the basic steps required for connecting to the YugabyteDB database using Laravel framework. The full working app using Laravel framework can be found in the [PHP ORM example application](../../orms/php/ysql-laravel/) page.
+
+### Step 1: Create a Laravel project
+
+```txt
+composer create-project --prefer-dist laravel/laravel yb-laravel-example
+```
+
+Note: Review the [prerequisites](../../orms/php/ysql-laravel//#prerequisites) for working with a Laravel project.
+
+### Step 2: Configure the datasource in Laravel project
+
+Update the `.env` file for configuring the Laravel project to connect to the YugabyteDB cluster.
```

Comment:
It's better to explicitly mention that the `.env` **inside the** `yb-laravel-example` directory needs to be modified as below.